tide_test.permissions.yml

```diff
@@ -0,0 +1,7 @@
+'view scheduled transitions node test':
+  title: 'View scheduled transitions for node test'
+  description: 'View all scheduled transitions for test content type.'
+
+'add scheduled transitions node test':
+  title: 'Add scheduled transitions for node test'
+  description: 'Add scheduled transitions for test content type.'
```

Comment:
Do we have new tests that depend on this permission being set for the test user?

Reply:
This hook triggers when a content type gets created https://github.com/dpc-sdp/tide_core/blob/develop/tide_core.module#L321
Although I have removed it for D10 branch, because the permission needs to be available first by the content type module. This is the reason I added this here, so if any tests needs for test content, this module will be providing the permission for test content type.